### PR TITLE
Fix install instructions

### PR DESCRIPTION
**Pull request by hhdewarren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,13 +59,14 @@ $ source .venv/bin/activate
...
 ```bash
+$ python3 -m pip install --upgrade pip
 $ pip install --upgrade setuptools
 ```
 
-5. Install TensorFlow 2.2 (community-built wheel)
+5. Install TensorFlow 2.4 (community-built wheel)
 
 ```bash
-$ pip install https://github.com/leigh-johnson/Tensorflow-bin/releases/download/v2.2.0/tensorflow-2.2.0-cp37-cp37m-linux_armv7l.whl
```
Comment (hhdewarren) [inline README.md:68]: The link was not valid anymore

File: README.md [hunk 2]
```diff
@@ -80,7 +81,7 @@ pip install rpi-deep-pantilt
 NOTE: This step is only required if you are using [Coral's Edge TPU USB Accelerator](https://coral.withgoogle.com/products/accelerator). If you would like to run TFLite inferences using CPU only, skip this step. 
 
 ```bash
-$ pip install https://dl.google.com/coral/python/tflite_runtime-2.1.0.post1-cp37-cp37m-linux_armv7l.whl
```
Comment (hhdewarren) [inline README.md:83]: The version was too old.

Comment (hhdewarren) [reply README.md:83]: cf https://github.com/google-coral/edgetpu/issues/259

**Comment by leigh-johnson** [comment]
:bowing_woman: Thank you for fixing this @hhdewarren!

**Pull request merged**